_posts/2019-06-07-python-datascience-with-openfaas.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Serving Data Science Models with OpenFaaS
+description: "Lucas will build and deploy a name classification function using OpenFaaS and PyTorch"
+date: 2019-06-01
+image: /images/pydatascience-template/artificial-neural-network-3501528_640.png
+categories:
+  - python
+  - coding
+  - examples
+  - datascience
+  - templates
+author_staff_member: lucas
+dark_background: false
+---
+
```

Comment:
"Lucas will build and deploy a name classification function using OpenFaaS and PyTorch"